components/newsletter-sign-up/newsletter-sign-up.jsx

```diff
@@ -82,13 +82,18 @@ class NewsletterSignUp extends React.Component {
 
     if (email && this.validatesAsEmail(email) && consent) {
       this.submitDataToApi();
+      Analytics.ReactGA.event({
```

Comment:
Moved this within the if statement instead of outside, that way this will only run if the email is filled out as well as the consent box!